patches/chrome-VERSION.patch

```diff
@@ -8,4 +8,4 @@ index 3d9a560e26e43fb2da1332dff339c1039cfb1a3e..4050aa4ee4122124f154ceeff1e3ac83
 -BUILD=3683
 -PATCH=75
 +BUILD=65
-+PATCH=28
++PATCH=0
```

Comment:
To-Do: Remove. Changed to get around CI build issues, will be fixed in master soon.